ResearchKit/Common/ORKStepViewController.m

```diff
@@ -284,6 +284,18 @@ - (BOOL)hasBeenPresented {
     return _hasBeenPresented;
 }
 
+- (BOOL)canChangeStepResult {
+    if (!self.step || !self.taskViewController) {
+        return NO;
+    }
+    BOOL isReviewStepStandalone = NO;
+    ORKReviewStep *reviewStep = [self.taskViewController reviewStepForStep:self.step];
+    if (reviewStep) {
+        isReviewStepStandalone = reviewStep.isStandalone;
+    }
+    return self.step.isBeingReviewed ? !isReviewStepStandalone : YES;
```

Comment:
The taskVC already assigns `isBeingReviewed` to the step. I will change that property to belong to the stepVC as you suggested above.

If we let the taskVC assign `canChangeStepResult` as well, we should take care of proper state restoration. I think it can be done but won't make things a lot easier. However, the present approach might be a little easier to read for anyone who's trying to understand what's going on.